### PR TITLE
Simple querying API to Model

### PR DESCRIPTION
**Pull request by lietu**
I want to do a filter of finding datetimes in the past, it wasn't possible, so I added this.

### CODE REVIEW

**Review by joakimnordling — CHANGES_REQUESTED** [review]
- I'd rather see the `find` method extended to support something like `.find({"count": {">=": 3}})` than a new `query` method being added. `find` could be extended to work similarly to how it's done in [Arangodantic](https://github.com/digitalliving/arangodantic) or in [MongoDB](https://docs.mongodb.com/manual/reference/method/db.collection.find/). I think that approach is rather nice and extensible also in the future if/when we need to add support for more features.
- Please update the bottom section of the CHANGELOG.md; the link for Unreleased and add a 0.1.4 section to it, also update the release date once this is approved.
- It would be nice to have examples of use of the new functionality in the README.
- Please also add unit tests for new functionality.
- Also please check why Travis is failing.

File: firedantic/models.py [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
-from google.cloud.firestore_v1 import CollectionReference, DocumentReference
+from google.cloud.firestore_v1 import CollectionReference, DocumentReference, Query
 
 from firedantic.configurations import CONFIGURATIONS
 from firedantic.exceptions import CollectionNotDefined, ModelNotFoundError
 
 TModel = TypeVar("TModel", bound="Model")
 
+# https://firebase.google.com/docs/firestore/query-data/queries#query_operators
+FIND_TYPES = (
```
Comment (joakimnordling) [inline firedantic/models.py:13]: I would prefer a set for this kind of things where you do a lot of membership (something in something) checks.

File: firedantic/models.py [hunk 2]
```diff
@@ -49,10 +63,24 @@ def find(cls: Type[TModel], filter_: dict) -> List[TModel]:
         query = coll
```
Comment (joakimnordling) [inline firedantic/models.py:63]: A few rows up there's still a comment saying `Currently only supports `==` operator.`; could you please remove that?

Comment (joakimnordling) [reply firedantic/models.py:63]: Maybe also add another example with the operators there at the same time.

Comment (lietu) [reply firedantic/models.py:63]: Done

**Review by joakimnordling — APPROVED** [review]
Looks good!

**Comment by lietu** [comment]
Trying to migrate from travis-ci.org to travis-ci.com to get the builds working at all, but there are some issues with Travis I'm trying to get resolved.

**Pull request merged**